<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createPushButton">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>80</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Ramdisk</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rListPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramdisk List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitPushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Ramdisk</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sizeLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sizeHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramdisk Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="mountLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ramdisk Mount Point</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mountLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
